webcomponents-loader.js

```diff
@@ -23,26 +23,30 @@
   if (!window.customElements || window.customElements.forcePolyfill) {
     polyfills.push('ce');
   }
-  if (!('content' in document.createElement('template')) || !window.Promise ||
+  // NOTE: any browser that does not have template or ES6 features
+  // must load the full suite (called `lite` for legacy reasons) of polyfills.
+  if (!('content' in document.createElement('template')) || !window.Promise || !Array.from ||
     // Edge has broken fragment cloning which means you cannot clone template.content
     !(document.createDocumentFragment().cloneNode() instanceof DocumentFragment)) {
-    polyfills.push('pf');
-  }
-
-  if (polyfills.length === 4) { // hi-ce-sd-pf is actually called lite.
     polyfills = ['lite'];
   }
 
   // NOTE: we stub HTMLImports to enable WCT to wait for "WebComponentsReady" event.
-  window['HTMLImports'] = {};
```

Comment:
We were discussing changing [this line](https://github.com/Polymer/web-component-tester/blob/master/browser.js#L61) in WCT to `if (!window.HTMLImports && !document.querySelector('script[src*="webcomponents-"]')` (basically, loading any variant of `webcomponents-*.js` should guarantee that `WebComponentsReady` is fired.

@sorvell Was the intention to go forward and propose that change along with removal of the `HTMLImports` stub object above?